python/lsst/atmospec/formatters.py

```diff
@@ -31,10 +33,27 @@ class SpectractorSpectrumFormatter(FileFormatter):
     unsupportedParameters = None
 
     def _readFile(self, path, pytype=None):
-        return Spectrum(path)
+        spectrogramPath = path + 'spectrogram.fits'
```

Comment:
Does this yield file names like `/path/to/spectrum.ext`, `/path/to/spectrum.extspectrogram.fits`, and `/path/to/spectrum.exttable.csv`?